scripts/helmer.sh

```diff
@@ -74,6 +74,26 @@ mtls_configmap() {
     kubectl get cm -n $NAMESPACE mtlscm &>/dev/null || kubectl create cm -n $NAMESPACE mtlscm --from-file=client-ca.crt=$(dirname "$0")/../resources/mtls/rootCA.crt
 }
 
+create_kw_namespace_with_psa() {
+    local ns_name="$1"
+    enforce_version=$(kubectl version --short | grep "Server Version" | awk '{print $3}')
```

Comment:
```
~ kubectl version --short
error: unknown flag: --short

~ kubectl version 
Client Version: v1.32.2
Kustomize Version: v5.5.0
Server Version: v1.31.5+k3s1
```

Reply:
woops, Fixed it. Also, one needs to cut `v1.31.5+k3s1` to `v1.31`, which I did with a followup awk call.